VideoAPI/VideoApi.DAL/Migrations/20200205131422_AddMessage.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace VideoApi.DAL.Migrations
+{
+    public partial class AddMessage : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "Message",
+                columns: table => new
+                {
+                    Id = table.Column<long>(nullable: false)
+                        .Annotation("SqlServer:Identity", "1, 1"),
```

Comment:
Hi Sunil, I don't think any of these fields should allow null

From = table.Column(nullable: true),
To = table.Column(nullable: true),
MessageText = table.Column(nullable: true),
TimeStamp = table.Column(nullable: false),
ConferenceId = table.Column(nullable: true)

Reply:
Done